sdk/src/androidTest/java/com/mapbox/maps/OfflineTest.kt

```diff
@@ -36,25 +35,31 @@ class OfflineTest {
       )
     }
   }
+  private val resourceOptions = ResourceOptions.Builder().applyDefaultParams(
+    InstrumentationRegistry.getInstrumentation().targetContext
+  ).tileStore(tileStore).build()
   private val offlineManager: OfflineManager by lazy {
-    OfflineManager(
-      ResourceOptions.Builder().applyDefaultParams(
-        InstrumentationRegistry.getInstrumentation().targetContext
-      ).tileStore(tileStore).build()
-    )
+    OfflineManager(resourceOptions)
   }
 
   @get:Rule
   var rule = ActivityScenarioRule(EmptyActivity::class.java)
 
   @Before
   fun setUp() {
+    val latch = CountDownLatch(1)
     handler.post {
       offlineManager.removeStylePack(STYLE)
       tileStore.removeTileRegion(TILE_REGION_ID)
       tileStore.setOption(TileStoreOptions.DISK_QUOTA, Value(0))
-      // Fixme add method to clear cached data(style pack)
-      File("${InstrumentationRegistry.getInstrumentation().targetContext.filesDir.absolutePath}/.mapbox/map_data/map_data.db").delete()
+      MapboxMap.clearData(resourceOptions) {
+        if (!it.isError) {
+          latch.countDown()
+        }
+      }
+    }
+    if (!latch.await(30, TimeUnit.SECONDS)) {
```

Comment:
Maybe 30 is too much here?

Reply:
Most of the time the 30 does not really matter, it will finish really fast. 30 is just a value we used for all the latches.